Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/WriteProjectInfoFileTargetTests.cs

```diff
@@ -18,660 +18,422 @@
-
         var sqTargetFile = TestUtils.EnsureAnalysisTargetsExists(TestContext);
-        var projectFilePath = Path.Combine(rootInputFolder, "project.txt");
+        var projectFilePath = Path.Combine(TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext, "Inputs"), "project.txt");
         var projectGuid = Guid.NewGuid();
-
-        var projectXml = $@"<?xml version='1.0' encoding='utf-8'?>
-<Project ToolsVersion='12.0' xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
-
-  <PropertyGroup>
-    <ProjectGuid>{projectGuid}</ProjectGuid>
-
-    <SonarQubeTempPath>{rootOutputFolder}</SonarQubeTempPath>
-    <SonarQubeBuildTasksAssemblyFile>{typeof(WriteProjectInfoFile).Assembly.Location}</SonarQubeBuildTasksAssemblyFile>
-    <SonarQubeOutputPath>{rootOutputFolder}</SonarQubeOutputPath>
-  </PropertyGroup>
-
-  <Import Project='{sqTargetFile}' />
-</Project>
-";
+        var projectXml = $"""
+            <?xml version='1.0' encoding='utf-8'?>
+            <Project ToolsVersion='12.0' xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
+              <PropertyGroup>
+                <SonarQubeTempPath>{rootOutputFolder}</SonarQubeTempPath>
+                <ProjectGuid>{projectGuid}</ProjectGuid>
+                <SonarQubeOutputPath>{rootOutputFolder}</SonarQubeOutputPath>
+                <SonarQubeBuildTasksAssemblyFile>{typeof(WriteProjectInfoFile).Assembly.Location}</SonarQubeBuildTasksAssemblyFile>
+              </PropertyGroup>
+              <Import Project='{sqTargetFile}' />
+            </Project>
+            """;
         var projectRoot = BuildUtilities.CreateProjectFromTemplate(projectFilePath, TestContext, projectXml);
 
-        // Act
-        var result = BuildRunner.BuildTargets(TestContext, projectRoot.FullPath,
-            TargetConstants.SonarWriteProjectData);
+        var result = BuildRunner.BuildTargets(TestContext, projectRoot.FullPath, TargetConstants.SonarWriteProjectData);
 
-        // Assert
         result.AssertTargetSucceeded(TargetConstants.SonarWriteProjectData);
-
         var projectInfo = ProjectInfoAssertions.AssertProjectInfoExists(rootOutputFolder, projectRoot.FullPath);
-
         projectInfo.ProjectGuid.Should().Be(projectGuid, "Unexpected project guid");
         projectInfo.ProjectLanguage.Should().BeNull("Expecting the project language to be null");
         projectInfo.IsExcluded.Should().BeFalse("Project should not be marked as excluded");
         projectInfo.ProjectType.Should().Be(ProjectType.Product, "Project should be marked as a product project");
         projectInfo.AnalysisResults.Should().BeEmpty("Not expecting any analysis results to have been created");
     }
 
-    [TestCategory(TestCategories.NoUnixNeedsReview)]
+    // Checks the WriteProjectInfo target handles projects with unrecognized languages
+    // As this specifically tests projects that dont use C#/VB we dont use TargetsTestsContext
```

Comment:
```suggestion
    // Checks the WriteProjectInfo target handles projects with unrecognized languages.
    // As this specifically tests projects that don't use C#/VB we don't use TargetsTestsContext.
```
Without punctuation, this looks like a run-on sentence.